doc/user/content/release-notes.md

```diff
@@ -54,6 +54,13 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.5.1 %}}
 
+- **Known issue.** [`COPY TO`](/sql/copy-to) panics if executed via the
+  ["simple query" protocol][pgwire-simple], which is notably used by the `psql` command-line client {{% gh 4742 %}}.
+
+  A fix is available in the latest [unstable builds](/versions/#unstable-builds)
+  and will ship in v0.5.2. In the meantime, you can use a PostgreSQL driver that supports the ["extended query" protocol][pgwire-extended], and use only the
+  extended query protocol to issue `COPY TO` statements.
```

Comment:
While this is a technically true statement, I'm not sure there is a driver that is even able to do this, so it may be completely not useful to users. I...don't know if or how we should change what you have written though. Pointing people toward 0.5.2 is probably the best, which you've already done? I guess leave this?

Reply:
Oh, really? The Rust driver does by default (hence our passing tests), but now that you mention it I guess it could be the only one.

Reply:
Good point. Maybe a more accurate description then is: most drivers don't let you choose how COPY is sent, so if it doesn't work then there's probably nothing they can do until 0.5.2

Reply:
Ack, updated accordingly.